Jakefile

```diff
@@ -1191,7 +850,7 @@ task('_build-file', { async: true }, function() {
       ].join(' ')
     ], EXEC_OPTS);
     _getCommintHash.on('stdout', function(COMMIT_INFO) {
-        var BUILD_TIME = _extended_build_time(),
+        var BUILD_TIME = _extended_build_time();
```

Comment:
`COMMIT_INFO` is already defined as the function parameter, thank god for the linter ;)